ncm-nss/src/test/resources/simple.pan

```diff
@@ -0,0 +1,15 @@
+object template simple;
+
+#function pkg_repl = { null; };
```

Comment:
```
user-defined validation failed
element path: '/software/components/nss/version'
element value: 16.2.1-SNAPSHOT
```

because the [component type](https://github.com/quattor/template-library-core/blob/master/quattor/types/component.pan#L13) only allows digits in this field.